modules/ROOT/pages/correlation-id.adoc

```diff
@@ -0,0 +1,253 @@
+----
+
+=== Correlation ID Logs in DEBUG Level
+
+In case you want to see additional logs related to the correlation ID, you can set the log level to `DEBUG` by adding the following code to your application XML:
+
+----
+<AsyncLogger name="org.mule.runtime.module.extension.internal.runtime.operation" level="DEBUG"/>
+----
+
+After performing a request to the application from the previous example, the updated logs show as following:
+
+----
+WARN  2021-03-30 16:58:29,545 [[MuleRuntime].uber.05: [test-project-app].example.CPU_LITE @6d3b5ad] [processor: example/processors/1; event: bad0e5b0-9191-11eb-a0b3-36548d51aeee] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: some
+DEBUG 2021-03-30 16:58:29,548 [[MuleRuntime].uber.05: [test-project-app].example.CPU_LITE @6d3b5ad] org.mule.runtime.module.extension.internal.runtime.operation.ImmutableProcessorChildContextChainExecutor: Changing event correlationId from '4c355b30-9192-11eb-b79f-36548d51aeee' to '4c355b30-9192-11eb-b79f-36548d51aeee-EXAMPLE' in location example/processors/2
+WARN  2021-03-30 16:58:29,555 [[MuleRuntime].uber.05: [test-project-app].example.CPU_LITE @6d3b5ad] [processor: example/processors/2/processors/0; event: bad0e5b0-9191-11eb-a0b3-36548d51aeee-EXAMPLE] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: some
+DEBUG 2021-03-30 16:58:29,566 [[MuleRuntime].uber.05: [test-project-app].example.CPU_LITE @6d3b5ad] org.mule.runtime.module.extension.internal.runtime.operation.ImmutableProcessorChildContextChainExecutor: Event with correlationId '4c355b30-9192-11eb-b79f-36548d51aeee-EXAMPLE' going back to '4c355b30-9192-11eb-b79f-36548d51aeee' (successful execution) in location example/processors/2
+WARN  2021-03-30 16:58:29,568 [[MuleRuntime].uber.05: [test-project-app].example.CPU_LITE @6d3b5ad] [processor: example/processors/3; event: bad0e5b0-9191-11eb-a0b3-36548d51aeee] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: some
+----
+
+== Correlation ID Scope and Error Handling
+
+In case of an error, the error handler logs the correlation ID that corresponds to the flow or Try scope that contains the error handler. For example:
+
+[source,xml,linenums]
+----
+<flow name="example">
+  <tracing:with-correlation-id correlationId="#[correlationId ++ '-EXAMPLE']">
+    <raise-error type="APP:SOME"/>
+  </tracing:with-correlation-id>
+  <error-handler>
+    <on-error-continue>
+      <logger message="#[correlationId]"/>
+    </on-error-continue>
+  </error-handler>
+</flow>
+----
+
+In this scenario, Mule logs the correlation ID that the flow set at the beginning of the execution because the raised error stops the execution of the correlation ID scope, preventing the scope from setting the new correlation ID (`#[correlationId ++ '-EXAMPLE']`).
```

Comment:
```suggestion
In this scenario, Mule logs the correlation ID set at the beginning of the execution, because the error stops the execution of the correlation ID scope, preventing the it from setting the new correlation ID (`#[correlationId ++ '-EXAMPLE']`).
```